packages/melange.js/Js.ml

```diff
@@ -757,7 +757,29 @@ end = struct
     if start_idx >= end_idx then ""
     else Stdlib.String.sub str start_idx (end_idx - start_idx)
 
-  let split ?sep ?limit _str = notImplemented "Js.String" "split"
+  let split ?sep ?limit str =
+    match%platform () with
+    | Client -> Js.String.split ~sep ~limit str
+    | Server -> (
```

Comment:
This lib is only native, so no need to use platform ppx here

Reply:
😅 That make sense

Reply:
Done https://github.com/ml-in-barcelona/server-reason-react/pull/168/commits/cf71292de99b13fa9058fc72741a9608f11b1957